src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const inputHandle = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(e.currentTarget.value.trim());
+  };
+
+  const normalize = (text:string) => text.toLowerCase();
+
+  const filteredMovies = () => {
+    return [...moviesFromServer].filter(movie => {
+      return normalize(movie.title).includes(query.toLocaleLowerCase())
+        || normalize(movie.description).includes(query.toLocaleLowerCase());
```

Comment:
Can't you normalize the query using your function?

Reply:
I can, just I`ve seen some tutorial, they`ve sad we cant control the text we getting from user, inputs,forms....so I didn`t use it in that way. But now I fixed it)